doc-content/enterprise-only/employee-rostering/workbench-er-tutorial-con.adoc

```diff
@@ -0,0 +1,11 @@
+[id='workbench-er-tutorial-con']
+= Getting started with solvers in Workbench: An employee rostering example
+
+You can build and deploy the `employee-rostering` sample project in {CENTRAL}. The project  demonstrates how to create each of the {CENTRAL} assets required to solve the shift rostering planning problem and use {PLANNER} to find the best possible solution. 
+
+You can deploy the preconfigured `employee-rostering` project in {CENTRAL}. You can also create the project yourself using {CENTRAL}.
```

Comment:
"You can also create the project yourself using {CENTRAL}."
Does this mean this:
Alternatively, you can create your own project using {CENTRAL}.